<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GLWidget" name="glwidget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>520</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_z">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="x_axis_ScrollBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>0</height>
     </size>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="y_axis_ScrollBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="z_axis_ScrollBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_z">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_trans_model">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_z_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="x_axis_ScrollBar_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>0</height>
     </size>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="y_axis_ScrollBar_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="z_axis_ScrollBar_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_z_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_trans_model_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale Model</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="scale_ScrollBar">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>340</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_scale">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertexes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertexes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Edges:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vertexes">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_edges">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_trans_model_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Projection</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_parallel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_central">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_trans_model_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edges Type</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_solid">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_dashed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashed</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_trans_model_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Thickness</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_edge_color">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertexes_draw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertexes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_none">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_circle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_square">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_vertexes_size">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_vertex_size">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vertex_color">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vertex_color_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Background</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_screenshot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Screeshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gif">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
